inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -386,30 +386,20 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
 
     if (gnaFlags->sw_fp32) {
         gnamem->readonly().push_value(ptr_weights, power.scale, num_rows_out, 64);
-        gnamem->readonly().push_value(ptr_biases, power.scale, num_rows_out, 64);
+        gnamem->readonly().push_value(ptr_biases, power.offset, num_rows_out, 64);
     } else {
         auto weightsScaledIdentity = power.scale;
-        auto biasesScaledIdentity = power.scale;
+        auto biasesScaledIdentity = power.offset;
```

Comment:
how offset relates to scale?

Reply:
What do you mean?
Rename a variable, or how is biasesScaledIdentity related to power.offset?

Reply:
I would remove Scaled from biasesScaledEdentity - you can call it  just bias, also 
tehically this is not an Identity any mode - since it is scaled and shifted. Just use powerWeigths and powerBiases.

Also if quantized == 0 this code looks suspicious, any tests that hits quantized == nullptr? I think it cannot happen - we should properly quantize power layer in integer mode.